python/conda/meta.yaml

```diff
@@ -20,12 +19,15 @@ requirements:
   build:
     - python
     - cython >=0.13
+    - nomkl # [not win]
```

Comment:
Sounds good to me! The only issue I can see that might happen is when the nomkl and mkl numpy versions are different, but I am pretty sure that won't happen often in practice.